storage/storage_tests/storage_tests.cpp

```diff
@@ -700,6 +713,14 @@ UNIT_TEST(StorageTest_DownloadTwoCountriesAndDeleteTwoComponentMwm)
   TLocalFilePtr venezuelaFile = storage.GetLatestLocalFile(venezuelaCountryId);
   TEST(venezuelaFile.get(), ());
   TEST_EQUAL(MapOptions::Map, venezuelaFile->GetFiles(), ());
+
+  unique_ptr<CountryInfoGetter> countryInfoGetter = CreateCountryInfoGetter();
```

Comment:
Please, create separate tests for IsPointCovered, do not mix this code with existing tests.